### DISCUSSION TITLE
is it possible to use `proxy: redirect` while also masking the urls

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#129] [General]
is it possible to use `proxy: redirect` while also masking the URLs for security purpose?

**Comment by euzu** [comment]
@aniel300 currently not. But at the end if you use redirect the player  will get the link when it tries to open a stream.

**Reply by aniel300** [reply]
yes, but from a security standpoint the user and pass get leak and i don't like that

**Reply by euzu** [reply]
@aniel300 you can close this if its working.

**Reply by aniel300** [reply]
closing
